Overview.src.html

```diff
@@ -442,10 +442,38 @@ <h4 id=terminology-headers>Headers</h4>
 
 <h4>Bodies</h4>
 
-<p>A <dfn title=concept-body>body</dfn> is a byte stream. It has an associated
-<dfn title=concept-body-transmitted>transmitted</dfn> which is an integer and initially 0,
-<dfn title=concept-body-length>length</dfn> which is an integer and initially 0, and
-<dfn title=concept-body-error-flag>error flag</dfn> which is initially unset.
+<p>A <dfn title=concept-body>body</dfn> consists of a
```

Comment:
I think this would be slightly clearer like:

A **body** consists of:

- A **stream** (a ReadableStream object), initially null (?).
- A **transmitted** (an integer), initially 0.
- A **total bytes** (an integer), initially 0.

And maybe s/transmitted/transmitted bytes/g?